app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/AdditionalOptions.kt

```diff
@@ -127,9 +148,16 @@ fun AdditionalOptionSubMenu(
             }
 
             AdditionalOptionSubMenuState.Location -> {
-                LocationPickerComponent(
-                    onLocationPicked = onLocationPicked,
-                    onLocationClosed = onCloseAdditionalAttachment
+                FragmentHolderScreen(
+                    androidViewBindingFactory = FragmentContainerBinding::inflate,
+                    androidViewBindingUpdate = {
+                        with(fragmentContainerView.findFragment<DrawingFragment>()) {
+                            // Reference of AnalyticsEventsFragment is available here
+                        }
+                    },
+                    onBackPress = {
+
+                    },
```

Comment:
Please Ignore. This is removed in part 2.
This was part of the prev spike work.

Reply:
removed